terrain_planner/include/terrain_planner/terrain_ompl_rrt.h

```diff
@@ -69,9 +70,7 @@ class TerrainOmplRrt {
    *          - Negative: Clockwise
    */
   void setupProblem(const Eigen::Vector3d& start_pos, const Eigen::Vector3d& goal) {
-    this->setupProblem(
-        start_pos, goal,
-        problem_setup_->getStateSpace()->as<fw_planning::spaces::DubinsAirplaneStateSpace>()->getMinTurningRadius());
+    this->setupProblem(start_pos, goal, 66.6);
```

Comment:
Do you mean to hard code that value?

Reply:
Yes, getting the radius from the statespace is actually missing from the upstream implementation. I will extend upstream OMPL and at it back here

Reply:
Created a PR for the getter: https://github.com/ompl/ompl/pull/1285